e2e/pages/app/number-picker/updateOnBlur/number-picker-blur.testpage.component.html

```diff
@@ -0,0 +1,15 @@
+<a id="top-focus" href="javascript:void(0)">Top focus target</a>
+
+<div id="root">
+
+    <p>updateOnBlur (true):</p>
+    <ux-number-picker id="numberPicker1" class="number-picker-component-demo" min="-10" max="10" [formControl]="form.controls['updateOnBlur']"></ux-number-picker>
```

Comment:
Try to use meaningful id values, it makes working with them in the actual tests a lot easier.
```suggestion
    <ux-number-picker id="update-on-blur-number-picker" class="number-picker-component-demo" min="-10" max="10" [formControl]="form.controls['updateOnBlur']"></ux-number-picker>
```